crates/miden-tx/src/host/mod.rs

```diff
@@ -611,14 +623,15 @@ where
     /// Checks if the necessary witness for accessing the map item is already in the merkle store,
     /// and if not, extracts all necessary data for requesting it.
     ///
-    /// Expected stack state: `[KEY, ROOT, index]`
+    /// Expected stack state: `[event, KEY, ROOT, index]`
     pub fn on_account_storage_before_get_map_item(
         &self,
         process: &ProcessState,
     ) -> Result<TransactionEventHandling, TransactionKernelError> {
-        let map_key = process.get_stack_word(0);
-        let current_map_root = process.get_stack_word(1);
-        let slot_index = process.get_stack_item(8);
+        // => [signal, KEY, ROOT, index]
```

Comment:
Nit: We have the expected stack state in the doc comment already, so we can probably remove the one here in line 631.